packages/docs/src/pages/en/components/alerts.md

```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
 | [v-alert](/api/v-alert/) | Primary Component |
 | **v-alert-title** | Functional Component used to display the `v-alert` title. Wraps the `#title` slot |
 
-## Examples
+## Implementation
+
+Unlike the `v-banner` component, `v-alert` is intended to be re-used throughout your application as call-outs that draw a user's attention.
 
 ### Props
 
+In addition to the standard [v-sheet](/components/sheets/) properties such as elevation, dimension, and rounded, `v-alert` supports **v-model**, [variants](), and [density](); and many more.
+
 #### Type
 
-The **type** prop provides 4 default `v-alert` styles: **success**, **info**, **warning**, and **error**. Each of these styles provides a default icon and color. The default colors can be configured globally by customizing [Vuetify's theme](/features/theme).
+The `v-alert` component has 4 contextual states: **success**, **info**, **warning**, and **error**. Each state has a default _color_ and _icon_ associated with it. The following example uses the **type** prop to tell the `v-alert` that it's in an **error** state:
 
 <example file="v-alert/prop-type" />
 
-#### Border
+While the **type** prop shorthand is preferred, **color** and **icon** are usable individually and can achieve the same effect. The following example produces the same result as using **type="info"** by defining a custom color and using the icon lookup table to get the globally defined info icon:
 
-The **border** prop adds a simple border to one of the 4 sides of the alert. This can be combined with props like **color**, **dark**, and **type** to provide unique accents to the alert.
+```html
+<v-alert color="info" icon="$info">
+  I'm an info alert that uses <strong>color</strong> and <strong>icon</strong> props instead of <strong>type</strong>
+</v-alert>
+```
 
-<example file="v-alert/prop-border" />
+The `v-alert` component has multiple customization options such as reducing overall height using or adding a customized **border**. The following example uses [density]() to reduce the vertical space needed for the component:
 
-#### Colored border
+<example file="v-alert/prop-density" />
 
-The **colored-border** prop removes the alert background in order to accent the **border** prop. If a **type** is set, it will use the type's default color. If no **color** or **type** is set, the color will default to the inverted color of the applied theme (black for light and white/gray for dark).
+The **density** prop supports 3 levels of component height; **default**, **comfortable**, and **compact**.
 
-<example file="v-alert/prop-colored-border" />
+#### Variants
 
-#### Density
+`v-alert` has 6 style variants, **contained, contained-flat, contained-text, outlined, plain,** and **text**. By default, the `v-alert` component is **contained-flat**; which means that it has a solid background and does not have a box-shadow (elevation).
```

Comment:
I don't think we need this on every single component page that supports variants